nestkernel/histentry.h

```diff
@@ -40,20 +40,25 @@ class histentry
   double t_;              //!< point in time when spike occurred (in ms)
   double Kminus_;         //!< value of Kminus at that time
   double Kminus_triplet_; //!< value of triplet STDP Kminus at that time
-  size_t access_counter_; //!< access counter to enable removal of the entry, once all neurons read it
+  size_t
+    access_counter_; //! how often this entry was accessed (to enable removal, once read by all synapses which need it)
 };
 
-// entry in the history of plasticity rules which consider additional factors
+/**
+ * Class to represent a single entry in the spiking history of the ClopathArchivingNode or the UrbanczikArchivingNode.
+ *
+ * These history entries typically represent continuously-evolving values in time, so history timestamps correspond to
+ * ``nest.biological_time`` in simulation resolution rather than times of spikes.
+ */
 class histentry_extended
 {
 public:
   histentry_extended( double t, double dw, size_t access_counter );
 
-  double t_; //!< point in time when spike occurred (in ms)
+  double t_; //!< point in time for the history entry spike occurred (in ms)
```

Comment:
I think, the words "spike occurred" can be removed in this comment since it seems to be a remnant from the stdp documentation from where I copied it. Maybe one can refer to `t_` as "time stamp of the entry".

Reply:
Done!